registry/coder/modules/claude-code/main.tf

```diff
@@ -231,6 +236,42 @@ locals {
   start_script                      = file("${path.module}/scripts/start.sh")
   module_dir_name                   = ".claude-module"
   remove_last_session_id_script_b64 = base64encode(file("${path.module}/scripts/remove-last-session-id.sh"))
+
+  # Required prompts for the module to properly integrate with Coder
+  inner_system_prompt = <<-EOT
+      -- Tool Selection --
```

Comment:
I included only the `-- Tool Selection --` and `-- Task Reporting --` sections since they seem to be the only required for reliable Coder integration. In our dogfood we also have a `-- Framing --` and `-- Context --`, but those feel more org-specific, so I left them out. Let me know if you think it also makes sense to add those here as well